File: sqlInsert.py
import MySQLdb
#Open database connection
db =  MySQLdb.connect (host="localhost",user="User",passwd="password",db="school")
#prep a cursor object using cursor() method; cursor is like a navigation method
db.autocommit(True)
cursor = db.cursor(MySQLdb.cursors.DictCursor)

name='will'
age=15
gradeLevel=10

sql = (f"INSERT INTO students (name,age,gradeLevel) VALUES ('{name}',{age},{gradeLevel})")
cursor.execute(sql) 

cursor.close()
db.close()

File: sqlInsert.php

<?php

//require 'errorHandler.php';

$servername = "localhost";
$username = "willia20";
$password = "liang";
$dbname = "willia20";
//have to use single quote for variable
$name = 'will';
$age = 8;
$gradeLevel = 12;

//create connection
$conn = new mysqli($servername, $username, $password, $dbname);

// check connection
if($conn->connect_error){
    die("connection failed:".$conn->connect_error);
}

//inserts name,age grade into students table
$sql = "INSERT INTO students (name, age, gradeLevel) VALUES ('$name', '$age', '$gradeLevel')";

//creates a student record if not will display error
if ($conn->query($sql) === TRUE) {
    echo "New record created successfully";
} else {
    echo "Error: " . $sql . "<br>" . $conn->error;
}

$conn->close();




?>
